bls12_381/src/fp.rs

```diff
@@ -162,7 +162,7 @@ impl Fp {
         self.ct_eq(&Fp::zero())
     }
 
-    /// Attempts to convert a little-endian byte representation of
+    /// Attempts to convert a big-endian byte representation of
     /// a scalar into an `Fp`, failing if the input is not canonical.
     pub fn from_bytes(bytes: &[u8; 48]) -> CtOption<Fp> {
```

Comment:
I'd personally rather not; it implies a little-endian scalar encoding would also be correct and supported (rather than big-endian being canonical for `Fp`), and that way lies madness.